gravityflow/gravityflow
# Fix Order Summary always displaying with Complete Step Settings

## Description
Addresses [Issue #86](https://github.com/gravityflow/backlog/issues/86)
Order Summary always displays at end of Workflow, regardless of whether it is enabled or disabled on the Complete Step Settings.

## Testing instructions
1. Create a form with Pricing Fields.
2. Add a few workflow steps (like Approval, User Input, etc).
3. Edit Complete Step settings to disable Order Summary.
4. Process the form and workflow, at the end of the workflow the Order Summary would still be displayed.
5. Now switch to this branch, repeat step 4 and now Order Summary will not be displayed.
6. Update workflow to enable Order Summary on Complete Step and repeat step 4 again. Now Order Summary will display, as expected.

## Automated Test Enhancements
Added test `0052SummaryCompleteStepCept.php` to check for Order Summary Settings on Complete Step
 
## Screenshots
N/A

## Documentation Changes
N/A

## Checklist:
